test/setup/server_setup.go

```diff
@@ -181,12 +180,47 @@ func (s *Server) QueryData(t *testing.T, db, key string) (*types.GetDataResponse
 	return response, err
 }
 
-func (s *Server) WriteDataTx(t *testing.T, db, key string, value []byte) (string, *types.TxReceipt, error) {
-	client, err := s.NewRESTClient(s.clientCheckRedirect)
+func (s *Server) QueryUser(t *testing.T, userID string) (*types.GetUserResponseEnvelope, error) {
```

Comment:
How about also query user `alice` by a user other than `admin`? e.g. `alice` herself, or `bob`?
We can then check access control on the user db itself.

Reply:
This can be done in the subsequent PR.